### DISCUSSION TITLE
CRON job in api/routes

### DISCUSSION THREAD

**Discussion by bmbshlly** [vercel/community#951] [Help] [answered]
If you use AWS lambda for api/ in nextjs, aws supports cron job so why don't you which is no overhead in pricing?

**Comment by qodesmith** [comment]
Or, how can you achieve cron jobs with Vercel serverless functions? That's what I'm interested in.

**Comment by amyegan** [comment, marked as answer]
**Edit: Update 2023**

Wanted to share an exciting update – we just released Vercel Cron Jobs!

Vercel Cron Jobs enable you to run scheduled jobs for automating backups and archiving, sending email and Slack notifications, and more. Cron Jobs can be used for any task you need to run on a schedule.

https://vercel.com/blog/cron-jobs

**Reply by qodesmith** [reply]
Thanks Amy!
